### ISSUE TITLE
fig -> docker compose

### ISSUE THREAD

**Issue by tejasmanohar** [meanjs/mean#449]
https://github.com/meanjs/mean/tree/0.4.0#development-and-deployment-with-docker

fig is deprecated, docker compose is the new thing.

**Comment by ilanbiala** [comment]
Can you put together a PR to update it?

**Comment by kbowerma** [comment]
Yes but you can just change 'fig' to 'docker-compose' and it will even look for the fig.yml file.

However in the yml file you set the NODE_ENV to development which is looking at development.js which has localhost hardcoded to the uri. ( ```		uri: 'mongodb://localhost/mean-dev',```  I think you either need to make a new docker.js config  file that use ```uri: 'mongodb://db_1/mean-dev',``` or ```mean_db_1``` then set the ```NODE_ENV:  docker``` in the yml file.  

Or you could just edit the development.js to be something like:

```
uri:   process.env.DOCKER_MONGO_URI   ||  'mongodb://localhost/mean-dev',   

```
and set that var from the yaml file.

**Comment by SOSANA** [comment]
Once we fix this, can we update docs for others as well to get started with docker

**Comment by codydaig** [comment]
Has this been fixed with pull request #611 ?

**Comment by lirantal** [comment]
let's close, I think we're good here, otherwise @tejasmanohar can re-open.
